Automattic/jetpack
# Add Store ID in common properties.

Context: p7Ldg5-4O0-p2#comment-13058

## Proposed changes:

* This PR includes the new store_id property inside the default event properties which should be included with all events in woocommerce analytics package (soon to be migrated In core).

<img width="1548" alt="Screenshot 2024-08-13 at 15 39 10" src="https://github.com/user-attachments/assets/2b79c786-e11f-4fc1-8b76-5c3eb3dace21">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Install and Connect Jetpack. 
* Without being Admin navigate to a product
* Verify that store_id is being tracked 

Note that for testing I recommend to use [Tracks Vigilante ](https://github.com/Automattic/tracks-vigilante-2-chrome-extension)
